release.sh

```diff
@@ -25,9 +28,10 @@ echo "Executing: JAVA_OPTS=\"$JAVA_OPTS_VAL\" sbt \"$ARGS\""
 JAVA_OPTS="$JAVA_OPTS_VAL" ./sbtwrapper -DdockerUpLatest=${dockerUpdateLatest} "$ARGS"
 
 if [[ "$BACKPORT" == "true" ]]; then
-  echo "Backport release - Skipping updating docs (by push to master)"
+  echo "Backport release - Skipping updating master"
+elif [[ "$RC" == "true" ]]; then
+  echo "Release Candidate - Skipping updating master"
```

Comment:
Just asking, why for we use master branch ?

Reply:
because we want to have references to last stable branch in our docs and on docker hub (the `latest` tag points to that branch)